pandas/_libs/src/compat_helper.h

```diff
@@ -38,13 +38,9 @@ PANDAS_INLINE int slice_get_indices(PyObject *s,
                                     Py_ssize_t *stop,
                                     Py_ssize_t *step,
                                     Py_ssize_t *slicelength) {
-#if PY_VERSION_HEX >= 0x03000000
+
   return PySlice_GetIndicesEx(s, length, start, stop,
                               step, slicelength);
```

Comment:
if we're returning this unchanged, can we just use this function directly and not define an alias?  i guess the pypy thing above might complicate this?

Reply:
Unclear at this point, but makes sense to investigate in a follow-up.